### DISCUSSION TITLE
Are we going to migrate the view counter project to the new Astro site?

### DISCUSSION THREAD

**Discussion by stdavis** [agrc/gis.utah.gov#2727] [Announcements]
The [view counter project](https://github.com/agrc/view-counter) is what made this work in our old website:
![image](https://github.com/agrc/gis.utah.gov/assets/1326248/01242103-02f7-49e9-ac94-f193cc3504e2)

This screenshot is from the wayback machine which may be the cause of the strange view count value.

Do we want to migrate it to the new site? It's not a huge project but it probably needs some love if we want to keep it around. I'm not sure how I feel about it. I would probably drop it if it was up to me just to make less tech debt work. But if there is consensus that it's useful, I'm happy to make it happen.

**Comment by steveoh** [comment]
I'm happy to lose it.

**Comment by nathankota** [comment]
I _kind of_ liked seeing those, but if they're bogus and more tech debt I agree to trash it.

**Reply by stdavis** [reply]
The bogus data was only in the wayback machine. I do believe that they were giving good data. But yes, more tech debt. Another project for dependabot fun.
